### PR TITLE
Karma: use dots reporter

### LINKED ISSUE #8807: Karma: use dots reporter
## Task Description

Karma test output can be quite long.

Let's use a dot reporter like in all our other test suites (unit, PHP, e2e) to reduce output.

With that, successful tests will be simple dots (`.`), whereas failures can still display detailed output.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Change karma reporter to dots from spec. 

## Relevant Technical Choices

Remove spec reporter all together. 

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8807

### CODE REVIEW

File: packages/story-editor/src/index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export { default as FontContext } from './app/font/context';
 export { default as useLoadFontFiles } from './app/font/actions/useLoadFontFiles';
 export { default as StoryPropTypes } from './types';
 export { GlobalStyle } from './theme';
-export { default as theme } from './theme'; // @todo To be refactored.
+export { default as theme } from './theme'; // @todo To be refactored at some point.
```
Comment (swissspidy) [inline packages/story-editor/src/index.js]: 🤔

**Comment by swissspidy** [comment]
Can you convert this back to a draft again since it's WIP? Then we don't unnecessarily do Percy snapshots and things like that.

FWIW, we should update the reporters config in the config files instead of passing `--reporters`. We can remove `spec` and `karma-spec-reporter` entirely in favor of `dots`

File: packages/story-editor/src/index.js (hunk 1 above)
Comment (spacedmonkey) [reply packages/story-editor/src/index.js]: This was to force the karma test to run.

Comment (spacedmonkey) [reply packages/story-editor/src/index.js]: Removed now

**Comment by swissspidy** [comment]
Odd unit test failure: https://github.com/google/web-stories-wp/runs/3838941852

Does that happen locally as well? Might be a flakey test that's worth looking into in a separate ticket.

**Review by swissspidy — APPROVED** [review]

**Comment by spacedmonkey** [comment]
> Odd unit test failure:

Tests are now passing. Happy to merge?

**Pull request merged**